src/components/Resource/index.tsx

```diff
@@ -43,7 +43,7 @@ const Resource = ({ distributions, resource, title } : ResourcePropsType ) => {
             distributions.map((dist) => {
               const fileFormat = getFormatType(dist)
               return (
-                <li key={dist.identifier} className="ds-u-display--flex ds-u-flex-wrap--wrap">
+                <li key={dist.identifier} className={`ds-u-display--flex ds-u-flex-wrap--wrap ${fileFormat !== "csv" && "ds-u-margin--2"}`}>
```

Comment:
this should be `ds-u-margin-y--2`
It's adding a horizontal margin as well which we don't want

Reply:
@brdunfield feedback addressed

https://github.com/GetDKAN/cmsds-open-data-components/pull/181/commits/c306f567718d908d8317810d0f3a81334d473901